BUG Sensitivity plots with negative y-err fail with some matplotlib

## Summary of issue

Some of the plot tests are failing with newer matplotlib versions
```
tests/test_sensivity.py::SensitivityPlotTester::test_plot_labelFormatter FAILED [ 66%]
tests/test_sensivity.py::SensitivityPlotTester::test_plot_labelFormatter_oneIso FAILED [ 66%]
tests/test_sensivity.py::SensitivityPlotTester::test_plot_normalized FAILED [ 66%]
tests/test_sensivity.py::SensitivityPlotTester::test_plot_notNormalized_noSigma PASSED [ 66%]
tests/test_sensivity.py::SensitivityPlotTester::test_plot_passLabels_noset FAILED [ 67%]
...
tests/plots/test_sensitivity.py::test_sensitivity_filter FAILED          [ 99%]
tests/plots/test_sensitivity.py::test_sensitivity_kwargs FAILED          [100%]
```
with error message
```
E               ValueError: 'yerr' must not contain negative values
```
https://github.com/CORE-GATECH-GROUP/serpent-tools/actions/runs/3332679328/jobs/5513828100#step:5:700

This is happening for the python 3.8 tests, but 3.6 tests appear to be passing in #468. That job pulls [`matplotlib==3.3`](https://github.com/CORE-GATECH-GROUP/serpent-tools/pull/468/checks#step:4:13). Py3.7 job pulls [`matplotlib==3.5`](https://github.com/CORE-GATECH-GROUP/serpent-tools/actions/runs/3269472133/jobs/5377011025#step:4:15) while python 3.8 pulls [`matplotlib==3.6.1`](https://github.com/CORE-GATECH-GROUP/serpent-tools/actions/runs/3332679328/jobs/5513828100#step:4:15)

## Code for reproducing the issue

CI log file - https://github.com/CORE-GATECH-GROUP/serpent-tools/actions/runs/3332679328/jobs/5513828165

## Actual outcome including console output and error traceback if applicable
```
E               ValueError: 'yerr' must not contain negative values
```
## Expected outcome
Able to do sensitivity plots with newer matplotlib versions

## Versions

Please provide the following:

* Version  from ``serpentTools.__version__``
* Python version - ``python --version``
* IPython or Jupyter version if applicable
* If using the development version, the output from ``git describe``

--- a/serpentTools/parsers/sensitivity.py
+++ b/serpentTools/parsers/sensitivity.py
@@ -4,7 +4,7 @@ Class to read Sensitivity file
 from collections import OrderedDict
 from itertools import product
 
-from numpy import transpose, hstack
+from numpy import transpose, hstack, fabs
 from matplotlib.pyplot import gca
 
 from serpentTools.utils.plot import magicPlotDocDecorator, formatPlot
@@ -450,7 +450,7 @@ class SensitivityReader(BaseReader):
             yVals = values[iM, iZ, iP]
             yVals = hstack((yVals, yVals[-1]))
             yErrs = errors[iM, iZ, iP]
-            yErrs = hstack((yErrs, yErrs[-1]))
+            yErrs = fabs(hstack((yErrs, yErrs[-1])))
             label = labelFmt.format(r=resp, z=z, m=m, p=p)
             ax.errorbar(energies, yVals, yErrs, label=label, **kwargs)
 

